requirements.txt

```diff
@@ -0,0 +1,10 @@
+# #############################################################################
+# requirements
+# #############################################################################
+pywin32 >= 2.2.7; python_version > '2.7'
+# https://github.com/mhammond/pywin32
+# License: PSF
+
+pythonnet >= 2.4.0; python_version > '2.7'
+# https://github.com/pythonnet/pythonnet
+# License: MIT
```

Comment:
Remove this file.  This package has no `installs_requires`.  We can add this later.